docs/tutorial/add-subscription.md

```diff
@@ -51,14 +51,14 @@ from recipes_manager.data import RECIPES
 async def subscription_cooking_time(
     parent_result, args, ctx, info
 ):
-  recipe = [r for r in RECIPES if r["id"] == int(args["id"])]
+  recipe = next([r for r in RECIPES if r["id"] == int(args["id"])], None)
```

Comment:
Your code doesn't work.

The result of the list comprehension `[r for r in RECIPES if r["id"] == int(args["id"]` is not a valid iterator. You have to wrap the result with `iter()`.